cli/src/commands/packages.rs

```diff
@@ -35,22 +35,22 @@ pub async fn handle_get_package(api: &PhylumApi, matches: &clap::ArgMatches) ->
     };
 
     match resp {
-        PackageSubmitResponse::AlreadyProcessed(mut resp) => {
+        PackageSubmitResponse::AlreadyProcessed(mut resp) if resp.complete => {
```

Comment:
`resp.complete` should always be `true`. If it's false that's a bug.

Reply:
Well then there is a bug.

Reply:
Well then the API is broken.

Reply:
I can confirm that something is broken for some of these packages... We're still investigating the cause...

The `complete` field should probably be deprecated and removed. I don't think this is a CLI bug at all.

Reply:
Easy enough to remove in the future. For now this works around the API bug with just a single line changed.